packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-stages.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import type { ConnectedProps } from 'react-redux';
+import {
+  Pipeline,
+  Stage,
+  Description,
+  Link,
+  css,
+  cx,
+  spacing,
+  Button,
+  Icon,
+} from '@mongodb-js/compass-components';
+
+import type { RootState } from '../../../modules';
+import { stageAdded } from '../../../modules/pipeline';
+import { changeWorkspace } from '../../../modules/workspace';
+
+const containerStyles = css({
+  marginLeft: spacing[1],
+  marginRight: spacing[1],
+});
+
+const addStageStyles = css({
+  border: 'none',
+  backgroundColor: 'transparent',
+});
+
+const stagesStyles = css({
+  display: 'flex',
+});
+
+const PipelineStages: React.FunctionComponent<PipelineStagesProps> = ({
+  isEditing,
+  stages,
+  onStageAdded,
+  onChangeWorkspace,
+}) => {
+  if (stages.length === 0) {
+    return (
+      <Description className={containerStyles}>
+        Your pipeline is currently empty. To get started select the
+        <Link
+          className={addStageStyles}
+          as="button"
+          onClick={() => onStageAdded()}
+          hideExternalIcon
+        >
+          first stage.
+        </Link>
+      </Description>
+    );
+  }
+  return (
+    <div className={cx(containerStyles, stagesStyles)}>
+      <Pipeline size="small">
+        {stages.map((stage) => (
+          <Stage key={stage}>{stage}</Stage>
```

Comment:
Should we show some text when this is empty or filter those out? Currently looks a bit weird:

<img width="189" alt="image" src="https://user-images.githubusercontent.com/5036933/158763672-48aba6cf-b62e-4160-90e6-8bb371e5e185.png">

Reply:
I was thinking of filtering it out. Not sure. I'll check with Claudia what's better here.